swan-cern/files/swan_config_cern.py

```diff
@@ -251,5 +251,21 @@ def swan_pod_hook_prod(spawner, pod):
 
 c.SwanKubeSpawner.modify_pod_hook = swan_pod_hook_prod
 
+class SwanPostStopHookHandler:
+    def post_stop_hook(self, spawner):
+         # Delete Kubernetes Secret storing eos tokens
```

Comment:
Is this the EOS token exchanged with SSO? Only EOS or also CERNBox? If both I'd suggest, `Delete Kubernetes secret storing exchanged tokens for EOS and CERNBox`.

Reply:
OAuth tokens are downloaded directly from `/api/user` which also returns the `auth_state` stored in the jupyterhub database (custom handler implemented by [SwanHub](https://github.com/swan-cern/jupyterhub-extensions/blob/9d7e2e73992c29dcf5438e0ff7a92b1b4025a66d/SwanHub/swanhub/userapi_handler.py#L26)). 

The Kubernetes secret stores the EOS kerberos secret.